massalabs/massa-sc-playground
# 228 simulator integration updating export

Some limitations :
- Files url are not very relevant 

At this moment according with PM. We don't need to spend more time on this.
Process is working : Uploading Config file, Send it to Playground Plugin API , Receive Two files

Needs : 
The model config file imported in the repo (execution_config.json) should be respected  
The playground Plugin API is needed to launch simulator and handle requests

To test : 
Clone Playground plugin API , Run it 
Run npx http-server on playground
Upload config file 
Simulate
Verify JSON file on urls or In the Playground plugin API repo

Links : 
API PLUGIN : https://github.com/massalabs/thyra-playground-plugin
